src/init.cpp

```diff
@@ -528,6 +529,14 @@ static void BlockNotifyCallback(const uint256& hashNewTip)
     boost::thread t(runCommand, strCmd); // thread runs free
 }
 
+static void TxExpiryNotifyCallback(const std::string& txid)
+{
+    std::string strCmd = GetArg("-txexpirynotify", "");
+
+    boost::replace_all(strCmd, "%s", txid);
```

Comment:
... and then making this:
```suggestion
    boost::replace_all(strCmd, "%s", txid.GetHex());
```